gensim/models/translation_matrix.py

```diff
@@ -0,0 +1,329 @@
+#!/usr/bin/env python
+# encoding: utf-8
+import warnings
+import numpy as np
+
+from collections import OrderedDict
+from gensim import utils
+from six import string_types
+
+
+"""
+Produce translation matrix to translate the word from one language to another language, using either
+standard nearest neighbour method or globally corrected neighbour retrieval method [1].
+
+This method can be used to augment the existing phrase tables with more candidate translations, or
+filter out errors from the translation tables and known dictionaries [2].
+
+Initialize a model with e.g.::
+
+    >>> transmat = TranslationMatrix(word_pair, source_word_vec, target_word_vec)
+
+Train a model with e.g.::
+
+    >>> transmat.train(word_pair)
```

Comment:
This API that supplies `word_pair` twice - both at construction & a later necessary `train()` step – seems odd to me, and in contrast to other gensim practice (where training can be combined with initialization, if training data is supplied.)

Reply:
Done.